src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -77,6 +77,17 @@
       </_PackageValidationIntermediateBaselineFileContent>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(PackageValidationBaselineVersion)' == ''">
+      <_PackageValidationIntermediateBaselineFile>$(MSBuildProjectExtensionsPath)$(MSBuildProjectFile).packagevalidation.g.targets</_PackageValidationIntermediateBaselineFile>
+      <_PackageValidationIntermediateBaselineFileContent>
+<![CDATA[<Project>
+  <PropertyGroup>
+    <EnablePackageBaselineValidation>false</EnablePackageBaselineValidation>
```

Comment:
Will this have a potential race condition with the user's set of the property? One way to deal with that would be to do it in a target that runs before validation so that no race conditions can happen.

Reply:
i think we should be good here as i added it to the targets file which will be ported after the csproj file.

Reply:
you are right as long as they define it on the csproj file, but what if they define it in their Directory.Build.targets? My advice here in general would be that we put it in a target that runs right before we actually use it, so that we don't depend on any assumption of how people are structuring their projects.

Reply:
I wrote a target which runs before package validation and resets the EnablePackageBaselineValidation to false